fix_widget.py

```
@@ -0,0 +1,27 @@
+import os
+
+WIDGET_PATH = "dot_config/zsh/lazy/widget.zsh"
+
+with open(WIDGET_PATH, "r") as f:
+    lines = f.readlines()
+
+new_lines = []
+skip = False
+for i, line in enumerate(lines):
+    if "_git_mux()" in line:
+        if any("_git_mux()" in l for l in new_lines):
```

Ruff (0.15.7)

[error] Ambiguous variable name: `l`

(E741)